test/e2e/specs/specs-wpcom/wp-editor__post-basic-flow-spec.ts

```diff
@@ -119,7 +117,7 @@ describe( DataHelper.createSuiteTitle( 'Editor: Basic Post Flow' ), function ()
 			}
 		} );
 
-		it.skip( 'Close preview', async function () {
+		skipItIf( targetDevice === 'desktop' )( 'Close preview', async function () {
```

Comment:
That's a pretty neat guard pattern. Is that part of Jest?

Reply:
It's something we implemented ourselves in [this file](https://github.com/Automattic/wp-calypso/blob/trunk/packages/calypso-e2e/src/jest-conditionals.ts) :)